src/box/vy_cache.c

```diff
@@ -52,6 +52,24 @@ enum {
 	VY_CACHE_CLEANUP_MAX_STEPS = 10,
 };
 
+static void *
+vy_cache_tree_page_alloc(struct matras_allocator *allocator)
+{
+	(void)allocator;
+	void *ret = malloc(VY_CACHE_TREE_EXTENT_SIZE);
+	if (ret == NULL)
+		diag_set(OutOfMemory, VY_CACHE_TREE_EXTENT_SIZE, "malloc",
+			 "ret");
+	return ret;
```

Comment:
The checkpatch complains about this malloc call, which is copied from below to allow specify the functions on the matras allocator construction.

Replace with xmalloc?

Reply:
Done.

```diff
--- a/src/box/vy_cache.c
+++ b/src/box/vy_cache.c
@@ -56,11 +56,7 @@ static void *
 vy_cache_tree_page_alloc(struct matras_allocator *allocator)
 {
        (void)allocator;
-       void *ret = malloc(VY_CACHE_TREE_EXTENT_SIZE);
-       if (ret == NULL)
-               diag_set(OutOfMemory, VY_CACHE_TREE_EXTENT_SIZE, "malloc",
-                        "ret");
-       return ret;
+       return xmalloc(VY_CACHE_TREE_EXTENT_SIZE);
 }
 
 static void
```